docker/Dockerfile

```diff
@@ -53,6 +53,9 @@ RUN echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources
 RUN apt-get update
 RUN apt-get install yarn -y
 
+# Uncomment following line to force Docker to skip cache from here on:
+# ADD "https://www.random.org/cgi-bin/randbyte?nbytes=10&format=h" skipcache
+
```

Comment:
This is how we can force the build to not use the cached repo, but we can still get the benefit of having all the previous installation stuff cached.